os/Makefile.unix

```diff
@@ -519,11 +519,25 @@ post:
 ifeq ($(CONFIG_APP_BINARY_SEPARATION),y)
 	$(Q) mkdir -p $(OUTBIN_DIR)/user
 ifneq ($(CONFIG_SUPPORT_COMMON_BINARY),y)
+ifeq ($(CONFIG_APP_INFO),y)
```

Comment:
There are no APP_INFO configs in any Kconfig. How about adding them?

Reply:
In the ecode repo.. APP_INFO is added instead of APPx_INFO..

Reply:
What I want is unifying the code.
Could you check options below?
1. remove app_info and use app1_info only for ecode
2. make new config to select single app or multi apps
   if single app is selected, app_info, if multi apps, app1_info and app2_info

I prefer 1st option, if impossible, how about 2nd?
I think if we apply 1st or 2nd, we can manage the same code for both.

Reply:
I also think that 1st option is better.

Reply:
Yes, I agree with 1st option. 
In that case, is it ok to change ecode so that all places..
app -> app1
APP -> APP1 ?

Reply:
Yes. app1 and APP1 looks okay.

Reply:
Ok.. Modified accordingly. 
I will make changes for app->app1 in ecode and raise a separate PR.